rspace/src/main/scala/coop/rchain/rspace/history/Store.scala

```diff
@@ -34,7 +35,7 @@ object StoreInstances {
     for {
       env <- Sync[F].delay {
               Env
-                .create()
+                .create(PROXY_SAFE)
```

Comment:
Do you know what is performance impact of "safe buffers"?

Reply:
https://github.com/rchain/rchain/issues/3127